.circleci/config.yml

```diff
@@ -176,6 +176,43 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  mcp-e2e-mixer-noauth-v1alpha3-v2:
```

Comment:
The "unofficial" naming convention here is e2e-{component}-{features}.

Could you update the new names to follow that too?

Reply:
Done

FYI, these separate MCP test variants are temporary until MCP is vetted and becomes the only option (~1.2?)